nuxt.config.js

```diff
@@ -76,6 +76,7 @@ export default {
           success: colors.green.accent3,
           background: '#f1f3f5', // same as $gray1
           gray9: '#212529', // same as $gray9
+          primary2:'#1669BB'
```

Comment:
It wouldn't hurt to say this is the same as $app-blue.

Reply:
updated

Reply:
The property name is self-documenting! :)

This conversation can be resolved.